examples/shell/pwsh.tape

```diff
@@ -0,0 +1,8 @@
+Output examples/shell/pwsh.gif
+Output examples/shell/pwsh.ascii
+
+Set Shell "pwsh"
+
+Type 'echo "current shell: $ShellId"'
```

Comment:
For PowerShell, it's more idiomatic to either write the string (without `echo` beforehand), or use `Write-Output` here. I would lean towards the latter for better optics with PowerShell users.